extensions/azurePublish/src/components/ChooseProvisionAction.tsx

```diff
@@ -0,0 +1,264 @@
+      <Summary>
+        <Text>
+          {formatMessage(
+            'Select this option when you want to provision new Azure resources and publish a bot. A subscription to Microsoft Azure is required.'
+          )}
+        </Text>
+      </Summary>
+      <Details>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 1')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>{formatMessage('Sign in to Azure.')}</Text>
+          </InstructionDetails>
+        </Instruction>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 2')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>
+              {formatMessage(
+                'Select tenant & subscription, enter resource group name and resource name, and select region.'
+              )}
+            </Text>
+          </InstructionDetails>
+        </Instruction>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 3')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>
+              {formatMessage(
+                'Review & create new resources. Once provisioned these resources will be available in your Azure portal.'
+              )}
+            </Text>
+          </InstructionDetails>
+        </Instruction>
+      </Details>
+      <LearnMoreLink>{formatMessage('Learn More')}</LearnMoreLink>
+    </Content>
+  );
+};
+
+const ImportActionContent = () => {
+  return (
+    <Content>
+      <Title>{formatMessage('Import existing resources')}</Title>
+      <Summary>
+        <p>
+          <Text>{formatMessage('Select this option to import existing Azure resources and publish a bot.')}</Text>
+        </p>
+        <p>
+          <Text>
+            {formatMessage(
+              'Edit the JSON file in the Publish Configuration field. You will need to find the values of associated resources in your Azure portal. A list of required and optional resources may include:'
+            )}
+          </Text>
+        </p>
+      </Summary>
+      <Details>
+        <ResourceTitle>
+          <Text>Microsoft Application Registration</Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Azure Hosting</Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Microsoft Bot Channels Registration </Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Azure Cosmos DB </Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Application Insights </Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Azure Blob Storage </Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Microsoft Language Understanding (LUIS)</Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Microsoft QnA Maker </Text>
+        </ResourceTitle>
+      </Details>
+      <LearnMoreLink>{formatMessage('Learn More')}</LearnMoreLink>
```

Comment:
needs link

Reply:
I will add when design/docs has one. No idea what to put here.